src/app/dev/DevToys.Tools/Tools/EncodersDecoders/JsonWebToken/JsonWebTokenDecoderGuiTool.cs

```diff
@@ -0,0 +1,593 @@
+﻿using DevToys.Tools.Helpers.JsonWebToken;
+using DevToys.Tools.Models;
+using Microsoft.Extensions.Logging;
+
+namespace DevToys.Tools.Tools.EncodersDecoders.JsonWebToken;
+
+internal sealed partial class JsonWebTokenDecoderGuiTool
+{
+    /// <summary>
+    /// Define if we want to validate the token or not
+    /// </summary>
+    private static readonly SettingDefinition<bool> validateTokenSetting
+        = new(
+            name: $"{nameof(JsonWebTokenDecoderGuiTool)}.{nameof(validateTokenSetting)}",
+            defaultValue: false);
+
+    /// <summary>
+    /// Define if we want to validate the token issuer signing key or not
+    /// </summary>
+    private static readonly SettingDefinition<bool> validateTokenIssuerSigningKeySetting
+        = new(
+            name: $"{nameof(JsonWebTokenDecoderGuiTool)}.{nameof(validateTokenIssuerSigningKeySetting)}",
+            defaultValue: false);
+
+    #region TokenIssuers
+    /// <summary>
+    /// Define if we want to validate the token issuers or not
+    /// </summary>
+    private static readonly SettingDefinition<bool> validateTokenIssuerSetting
+        = new(
+            name: $"{nameof(JsonWebTokenDecoderGuiTool)}.{nameof(validateTokenIssuerSetting)}",
+            defaultValue: false);
+
+    /// <summary>
+    /// Define the issuers list
+    /// </summary>
+    private static readonly SettingDefinition<string> tokenIssuerSetting
```

Comment:
Same thing here. Is it risky to save this user input in a setting that is persisted on the hard drive?